reports/historical/_base_importer.py

```diff
@@ -71,6 +71,8 @@ def slurp_secondary_col_table(
             major_col_heading = canon(
                 ws.cell(column=icol, row=major_col_heading_row).value
             )
+            if not major_col_heading:
+                continue
```

Comment:
When would this normally happen?

Reply:
For some of the sheets, there are spaces in between the columns which stretch the entire height of the data, for those, we need to skip them entirely, which was why I added that line @kilemensi